src/draive/splitters/basic.py

```diff
@@ -33,28 +33,32 @@ def split_text(
 
 def _split(
     text: str,
-    separators: tuple[str, str] | str | None = None,
+    separators: Sequence[str] | str | None = None,
 ) -> tuple[str, list[str]]:
     # prepare initial splitter - default is new paragraph
-    splitter: str
-    alt_splitter: str
+    splitters: Sequence[str]
+
     match separators:
-        case (str(primary), str(secondary)):
-            splitter = primary
-            alt_splitter = secondary
+        case [*separators]:
+            splitters = separators
         case str(primary):
-            splitter = primary
-            alt_splitter = "\n"
+            splitters = [primary]
```

Comment:
I believe we should add "\n" to the list to keep previous behavior and to have some fallback eventually (unless we already have the "\n" 🤔